org/hypercerts/claim/claim.json

```diff
@@ -1,6 +1,6 @@
 {
   "lexicon": 1,
-  "id": "org.hypercerts.claim",
+  "id": "org.hypercerts.claim.claim",
```

Comment:
This double `claim` doesn't really make sense.  On Telegram it looks like we have at least initial consensus on `org.hypercerts.impact.*` so in this case it would be `org.hypercerts.impact.claim`.